doc/sphinx/data_notes.rst

```diff
@@ -513,6 +513,18 @@ optdone
 
 A list that indexes which elements of `atomcoords`_ represent converged geometries.
 
+optstatus
+---------
+
+A list of integers representing the status of each step in an optimisation. The possible optimisation statuses are defined in bit value notation to allow for coding for multiple states and are given by:
+
+    * ``OPT_UNKNOWN = 0b000 = 0`` is the default and means optimisation is in progress.
+    * ``OPT_NEW = 0b001 = 1`` is set for every new optimisation (e.g. PES, IRCs, etc.)
+    * ``OPT_DONE = 0b010 = 2`` is set for the last step of an optimisation that converged.
+    * ``OPT_UNCONVERGED = 0b100 = 4`` is set for every unconverged step (e.g. should be mutually exclusive with ``OPT_DONE``)
+
+So, to robustly check if step `i` has converged, one should check ``data.optstatus[i] & OPT_DONE`` instead of ``data.optstatus[i] == OPT_DONE``.
```

Comment:
```suggestion
So, to robustly check if step ``i`` has converged, one should check ``data.optstatus[i] & OPT_DONE`` instead of ``data.optstatus[i] == OPT_DONE``.
```
This is what the pre-commit check is complaining about; the single backticks are for links.

Reply:
Thanks! Updated!